README.md

```diff
@@ -1,30 +1,137 @@
+# Prerequisites
+
+[Playwright](https://playwright.dev/) requires [Node.js](https://nodejs.org/en/download/package-manager).
 
 # Installing Playwright
 
+Clone the repository and execute the following commands:
+
 ```bash
+cd playwright-runbook
 npm install
+npx playwright install
 ```
 
-# Set .env
+# Configuring Environment Variables
+
+Rename `.env.sample` to `.env`, and update the environment values.
+
+For local testing, set the URL to `localhost` or `candig.docker.internal`. All the credentials can be found in the `CanDIGv2/tmp/keycloak` folder.
 
-Rename `.env.sample` to `.env` and modify the values
+# Running Tests
 
-# Running the Test
+All the tests can be run by using:
 
 ```bash
-npx playwright test # Runs all tests in the default folder
-npx playwright test --workers=1 --repeat-each=1 # Recommended for UI tests; for performance testing, adjust the number of workers and repeats as needed 
-npx playwright test katsu # Runs a single test file 
-npx playwright test katsu -g "service-info" # Run a single test inside a test file
+npx playwright test
 ```
 
-# To switch between test folder
+However, it’s recommended to run each project separately with specific settings.
 
-In `playwright.config.ts`, comment out the default folder and specify the folder you want to use:
+### Running UI Tests
 
-```ts
-export default defineConfig({
-  // testDir: "./ui-tests",
-  testDir: "./performance-tests",
-  ...
+```bash
+npx playwright test --project=ui --workers=1          # Run all tests with a single worker
+npx playwright test --project=ui --ui                 # Run interactive mode
+npx playwright test summary                           # Run a specific file
+npx playwright test summary -g "diagnosis graph"      # Run a single test
 ```
+
+### Running API Tests
+
+```bash
+npx playwright test --project=api --workers=100 --repeat-each=100   # Adjust the number as needed
+npx playwright test tyk                                # Run a specific file
+npx playwright test tyk -g "tyk service-info"          # Run a single test
+```
+
+# Test Folders
+
+There are two main groups of tests, each organized into its own folder: `ui-tests` and `api-tests`.
+
+### UI Tests
+
+The `ui-tests` folder contains tests that simulate user interactions with the data portal. These tests verify basic frontend functionality, such as:
```

Comment:
```suggestion
The `ui-tests` folder contains tests that simulate user interactions with the data portal. They assume that the default test data from the integration tests is loaded into the running deployment before running, and complete tests based on being logged in as user2, who is authorized for the program `SYNTH_02`. These tests verify basic frontend functionality, such as:
```
Is the edit I made true?

Reply:
I used the admin credentials for the api_tests. I don’t think it will affect the summary page. Courtney and I discussed this briefly, and I think it’s better addressed in a separate PR.